tests/framework/installer/ceph_manifests_previous.go

```diff
@@ -37,8 +42,16 @@ func (m *CephManifestsPreviousVersion) Settings() *TestCephSettings {
 	return m.settings
 }
 
-func (m *CephManifestsPreviousVersion) GetCRDs(k8shelper *utils.K8sHelper) string {
-	return m.settings.readManifestFromGitHub("crds.yaml")
+func (m *CephManifestsPreviousVersion) GetCRDs(t *testing.T, k8shelper *utils.K8sHelper) string {
+	k8sVersion, err := k8sutil.GetK8SVersion(k8shelper.Clientset)
+	if err != nil {
+		assert.NoError(t, err)
+	}
+	minValidatinPolicy := k8sVersion.AtLeast(version.MustParseSemantic(MinValidatinPolicy))
+	if minValidatinPolicy {
```

Comment:
nit: I'd recommend it's more readable just to use this directly instead of use a variable.
```suggestion
	if k8sVersion.AtLeast(version.MustParseSemantic(minValidationPolicy) {
```

Reply:
this file reverted as this is used for previous version upgrades